fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/AccountTransfersApiResourceSwagger.java

```diff
@@ -39,13 +39,13 @@ static final class GetAccountTransfersFromOffice {
             private GetAccountTransfersFromOffice() {}
 
             @Schema(example = "1")
-            public Integer id;
+            public Long id;
             @Schema(example = "HO")
             public String name;
             @Schema(example = "HO")
             public String nameDecorated;
             @Schema(example = "1")
-            public Integer externalId;
+            public Long externalId;
```

Comment:
This looks wrong to me...It should be **String** based on the OfficeData.java line 37 (it is ExternalId type, but we use the String value in the fineract-client)

Reply:
Done.